src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8814.cs

```diff
@@ -17,14 +16,9 @@ public Issue8814(TestDevice testDevice) : base(testDevice)
 
 		[Test]
 		[Category(UITestCategories.CollectionView)]
-		[Category(UITestCategories.Shell)]
-		[Category(UITestCategories.Compatibility)]
 		public void CollectionViewInShellShouldBeVisible()
 		{
-			this.IgnoreIfPlatforms([TestDevice.Android, TestDevice.iOS, TestDevice.Mac]);
-
-			App.WaitForNoElement(Success);
+			App.WaitForElement(Success);
```

Comment:
This change confused me, but I checked the original code and it's `WaitForElement`

![image](https://github.com/user-attachments/assets/7adb9a5f-127c-4cad-8f3a-00ffe26e886f)

So this must have been a bad copy paste